Frameworks/Social.Xaml/ShareSheet.xaml

```diff
@@ -0,0 +1,20 @@
+<ContentDialog
+    x:Class="Social.Xaml.ShareSheet"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Social.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Facebook"
+    PrimaryButtonText="Post"
+    SecondaryButtonText="Cancel"
+    PrimaryButtonClick="ShareSheet_Post"
+    SecondaryButtonClick="ShareSheet_Cancel">
+
+    <StackPanel>
+        <TextBlock Text="User Name" />
+        <TextBox PlaceholderText="Say something..." x:Name="Message" />
+        <Canvas Height="100" x:Name="Preview" />
```

Comment:
Or maybe yes? I tried letting the Canvas auto-size itself, but then it remains zero-size no matter what you put in it.

---
In reply to: [107996862](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107996862) [](ancestors = 107996862,107996042)